src/migration/ember-addon/steps/update-addon-package-json.js

```diff
@@ -65,13 +70,6 @@ function updateDevDependencies(packageJson, options) {
         'rollup-plugin-copy',
       ];
 
-  // May be easier to start over and add missing dependencies
-  devDependencies.clear();
-
-  packagesToDelete.forEach((packageName) => {
-    devDependencies.delete(packageName);
-  });
```

Comment:
Unused code. To be precise, a no-op because `devDependencies` becomes an empty map after line 69.